README.md

```diff
@@ -111,21 +119,28 @@ Right-click a package and select the `Export Package` option.  You can select wh
 
 <img src="/client/media/export_smart_contract_package.gif" width="85%" alt="Export a smart contract project">
 
-### Connect to local_fabric runtime
-The `local_fabric` runtime can be enabled in `Local Fabric Ops` panel. The first time you enable it, Fabric images will be installed.  
+### Connect to the local_fabric runtime
+
+**When using the pre-configured local instance of Hyperledger Fabric named `local_fabric`, the extension will automatically pull and use the correct Docker images.**
+
+The `local_fabric` runtime can be enabled in `Local Fabric Ops` panel. The first time you enable it, Fabric images will be installed and an admin identity created in the `local_fabric_wallet` visible in the `Fabric Wallets` panel. 
 Click the menu on the `Local Fabric Ops` header-bar for management options (including start/stop and teardown).
-Left-click the `local_fabric` identity to connect to it within the `Fabric Gateways` view.
+Left-click the `local_fabric` gateway to connect to it within the `Fabric Gateways` view, using the `admin` identity.
 
 <img src="/client/media/connect_local_fabric.gif" width="85%" alt="Connect to local_fabric runtime">
 
-### Teardown the local_fabric runtime
-When you start/stop the local_fabric, all data will be maintained. Choose `Teardown Fabric Runtime` from the `Local Fabric Ops` header-bar to completely teardown the runtime and start over (you must then confirm this action).
+### Connecting to your own Hyperledger Fabric instance and discover the existing resources
```

Comment:
`Connecting ... and discover` doesn't sound right - `Connect ... and discover` does - perhaps it should be `Connecting ... and discovering`?